Kconfig.nrf

```diff
@@ -18,8 +18,8 @@ config MAIN_STACK_SIZE
 	default 2048 if ZTEST
 	default 2048 if ENTROPY_CC3XX
 
-config ZTEST_STACKSIZE
-	default 2048 if ZTEST
+config TEST_EXTRA_STACKSIZE
```

Comment:
This will increase the stack size of all threads that make use of CONFIG_TEST_EXTRA_STACKSIZE in the tests suites. So it will potentially do more than increasing the stack size of the `ztest_thread`. Just FYI.